tests/src/test/javascript/crypto/EccCryptoContextSuite.js

```diff
@@ -0,0 +1,245 @@
+         },
+        privateKeyJSON: {
+            "kty": "EC",
+            "crv": "P-256",
+            "x":   "10jF0O9oX8GOXxxHgC9gktZbduHn0K3XI5EDum_fOkc",
+            "y":   "eBVHsIhtASlzUIzGnMAl0TDfj0pqgldZrbsZobEL-78",
+            "d":   "TBP2kKufnhTHEf88VPcOIPDlk8uAFlLgi7C1iv86huY",
+            "use": "sig",
+            "kid": "B"
+         }
+     };
+
+    /** Random. */
+    var random = new Random();
+    /** Message. */
+    var message = new Uint8Array(32);
+    random.nextBytes(message);
+    /** MSL context. */
+    var ctx = null;
+    var initialized = false;
+
+    beforeEach(function() {
+        var extractable = true;
+        var _algo = { name: "ECDSA", namedCurve: "P-256" };
+
+        if (!initialized) {
+            runs(function() {
+                mslCrypto["importKey"]("jwk", ECDSA_KEYPAIR_A.publicKeyJSON, _algo, extractable, ["verify"])
+                    .then(function (key) {
+                        publicKeyA = key;
+                    },
+                    function (err) {
+                        console.log(err);
+                    }
+                );
+                mslCrypto["importKey"]("jwk", ECDSA_KEYPAIR_A.privateKeyJSON, _algo, extractable, ["sign"])
+                    .then(function (key) {
+                        privateKeyA = key;
+                    },
+                    function (err) {
+                        console.log(err);
+                    }
+                );
+                mslCrypto["importKey"]("jwk", ECDSA_KEYPAIR_B.publicKeyJSON, _algo, extractable, ["verify"])
+                    .then(function (key) {
+                        publicKeyB = key;
+                    },
+                    function (err) {
+                        console.log(err);
+                    }
+                );
+                mslCrypto["importKey"]("jwk", ECDSA_KEYPAIR_B.privateKeyJSON, _algo, extractable, ["sign"])
+                    .then(function (key) {
+                        privateKeyB = key;
+                    },
+                    function (err) {
+                        console.log(err);
+                    }
+                );
+            });
+            waitsFor(function() { return publicKeyA && privateKeyA && publicKeyB && privateKeyB; }, "Import ECDSA keypairs A/B", 1500);
+            initialized = true;
+        }
+    });
+
+    describe("sign/verify", function() {
+
+         var oncomplete = function(pkey) {
+             return pkey;
+         };
+
+         var onerror = function(err) {
+             console.log(err);
+         }
+
+        it("sign/verify", function() {
+            var messageA = new Uint8Array(32);
+            random.nextBytes(messageA);
+
+            var messageB = new Uint8Array(32);
+            random.nextBytes(messageB);
+
+            var cryptoContext = new EccCryptoContext(ctx, privateKeyA, publicKeyA, EccCryptoContext$Mode.SIGN_VERIFY);
+            var signatureA = undefined, signatureB = undefined;
```

Comment:
For consistency, don't `= undefined`. It'll make global grep search/replace difficult in the future. (I used to `= undefined` but I think Mick suggested I do not.)